patchwork-user-profiles.md

```diff
@@ -1,299 +0,0 @@
-# Patchwork: User profiles
```

Comment:
It's probably worth keeping somewhere but I'm not sure if it really needs to be in the core docs.  @ahdinosaur I started to read the issue you link but it is really long and I'm not really sure you are intending it to illustrate.